source/math/FloatRandomGenerator.ooc

```diff
@@ -65,12 +75,24 @@ FloatGaussianRandomGenerator: class extends FloatRandomGenerator {
 		this _backend free()
 		super()
 	}
+	setRange: func (=_mu, =_sigma) {
+		_hasSecond = false
```

Comment:
The stored value (if any) was calculated for other choices of mean and standard deviation. If we change them, with this call, it is no longer correct.